### PR TITLE
Work and personal mails for user

### PR DESCRIPTION
**Pull request by nyet-ty**
## PR includes

-   [x] Feature

### CODE REVIEW

File: src/modules/serviceMethods.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const serviceMethods = {
             where: { serviceName_serviceId: { serviceName: data.serviceName, serviceId: data.serviceId } },
             include: { user: true },
         });
+        console.log('SERVICE_ID', data.serviceId);
```
Comment (tsumo) [inline src/modules/serviceMethods.ts:28]: log

File: src/modules/userMethods.ts [hunk 2]
```diff
@@ -647,6 +647,7 @@ export const userMethods = {
             where: { id: userCreationRequestId },
             include: { supplementalPositions: true, curators: true },
         });
+        console.log(request);
```
Comment (tsumo) [inline src/modules/userMethods.ts:650]: log

File: src/modules/userMethods.ts [hunk 3]
```diff
@@ -668,14 +669,17 @@ export const userMethods = {
         }
 
         if (request.personalEmail) {
-            const emailService = await prisma.externalService.findUnique({ where: { name: 'Email' } });
+            const emailService = await prisma.externalService.findUnique({ where: { name: 'PersonalEmail' } });
             if (!emailService) throw new TRPCError({ code: 'BAD_REQUEST', message: 'Email service not found' });
+
+            console.log(' if (request.personalEmail) {', emailService.name);
```
Comment (tsumo) [inline src/modules/userMethods.ts:675]: log

File: src/modules/userMethods.ts [hunk 4]
```diff
@@ -668,14 +669,17 @@ export const userMethods = {
...
             services.push({ serviceName: emailService.name, serviceId: request.personalEmail });
         }
 
         if (request.workEmail) {
-            const emailService = await prisma.externalService.findUnique({ where: { name: 'Email' } });
+            const emailService = await prisma.externalService.findUnique({ where: { name: 'WorkEmail' } });
             if (!emailService) throw new TRPCError({ code: 'BAD_REQUEST', message: 'Email service not found' });
+            console.log('if (request.workEmail) {', emailService.name);
```
Comment (tsumo) [inline src/modules/userMethods.ts:682]: log

**Review by tsumo — CHANGES_REQUESTED** [review]

**Review by tsumo — APPROVED** [review]

**Review by asabotovich — APPROVED** [review]

**Pull request merged**